### PR TITLE
add an option to disable stream option

### LINKED ISSUE #18: idea: set stream:false option to enable logging of API response
## What would you like to be added?
Option to set stream:false for Open AI API call.
Currently there is no good way to log API response when stream:true.
API response contains good information such as full text of completion, and token usage.

example:
    "usage": {
      "completion_tokens": 365,
      "prompt_tokens": 378,
      "total_tokens": 743
    }

## Why is this needed?
Being able to log full prompt and response text, as well as token usage is requested in some enterprise setting.

Sample implementation:
https://github.com/ks6088ts-labs/azure-openai-playground/compare/main...tkh85:azure-openai-playground:main

スクリーンショット 2023-06-09 220555

スクリーンショット 2023-06-09 221015

### PR DESCRIPTION
**Pull request by ks6088ts**
fixes #18

### CODE REVIEW

File: src/utils/OpenAI/OpenAI.ts [hunk 1]
```diff
@@ -5,13 +5,19 @@ import {
   ReconnectInterval,
 } from "eventsource-parser";
 
+// Enabled by default. Stream option is disabled only when AZURE_OPENAI_API_STREAM_ENABLED is set to `false`.
```
Comment (ks6088ts) [inline src/utils/OpenAI/OpenAI.ts:8]: To support backward compatibility, stream option is enabled by default.
Stream option is disabled only when users set `AZURE_OPENAI_API_STREAM_ENABLED=false` explicitly.

File: src/utils/OpenAI/OpenAI.ts [hunk 2]
```diff
@@ -80,9 +86,24 @@ export const getOpenAICompletion = async (
...
+        for await (const chunk of response.body as any) {
+          parser.feed(decoder.decode(chunk));
+        }
+      } else {
+        try {
+          const jsonData = (await response.json()) as any;
+          const text = jsonData.choices[0].message?.content;
+          const usage = jsonData.usage;
+          const systemMessage =
+            "[total tokens:" + String(usage.total_tokens) + "]";
```
Comment (ks6088ts) [inline src/utils/OpenAI/OpenAI.ts:100]: add total tokens message

**Review by tkh85 — APPROVED** [review]
Looks great!  Thoughtful implementation with backwards compatibility.

**Comment by ks6088ts** [comment]
I've just merged this to main branch.
Also I pushed tag `v0.0.9`, which triggered [release action](https://github.com/ks6088ts-labs/azure-openai-playground/actions/runs/5245087434/jobs/9472026954)
Now [ks6088ts/azure-openai-playground:0.0.9](https://hub.docker.com/layers/ks6088ts/azure-openai-playground/0.0.9/images/sha256-61192866400de6e7d0f62c1bd45ed3b2eb674fa6cb650104b66f3b74ff9022df?context=explore) is available on Docker Hub.

**Pull request merged**